build.gradle

```
@@ -21,7 +21,7 @@ allprojects {
         google()
         jcenter()
         maven {
-            url 'http://bitmovin.bintray.com/maven'
+            url 'https://bitmovin.jfrog.io/artifactory/public-releases'
```

The correct url would be

```suggestion
            url 'https://artifacts.bitmovin.com/artifactory/public-releases'
```

---

updated